fetch_system_config/root/lib/udev/rules.d/99-ds4drv.rules

```diff
@@ -0,0 +1,6 @@
+KERNEL=="uinput", MODE="0666"
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="054c", ATTRS{idProduct}=="05c4", MODE="0666"
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", KERNELS=="0005:054C:05C4.*", MODE="0666"
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="054c", ATTRS{idProduct}=="09cc", MODE="0666"
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", KERNELS=="0005:054C:09CC.*", MODE="0666"
+KERNEL=="js?", SUBSYSTEM=="input", ATTRS{name}=="Sony Computer Entertainment Wireless Controller", SYMLINK+="ps3joy"
```

Comment:
Are you symlinking a PS4 controller to a link named PS3? What happens if both connect?

Reply:
This is something I expected to be debatable.

My take is we should not bother trying to support a usecase where both are connected simultaneously. (plus, ps3joy + ds4drv may not be able to work simultaneously anyways; haven't tested).

If we make both udev rules create the same device, then we don't need to set an additional parameter to correctly set the /joy/dev ros param.

Currently fetch_teleop sets the /dev/ps3joy as the joy_device parameter.

Perhaps a better approach would be to (1) have udev rules create e.g. /dev/fetch_joy and (2) update the launch file for fetch_teleop with this new name.  This should work when someone does an upgrade, assuming they grab the latest fetch-melodic-config, too...

Reply:
I'm fine  with it. If you're using 2 different controllers you are doing something out of the ordinary. 

The change I think i'd like is 2 symlinks. One for ps3 one for ps4 so in theory there is a way to use both at the same time it's just weird and complicated

Reply:
Yeah, I can set two symlinks for both controller types :+1:

Reply:
This is now done and squashed into one of the commits :+1: